.github/workflows/publish-sdk.yml

```diff
@@ -58,22 +58,28 @@ jobs:
           ./scripts/gen-go-service-sdk.sh $SERVICE
           rm -rf $GITHUB_WORKSPACE/hcp-sdk-go/temp
 
+      - name: Create branch with updated service specs
+        run: |
+          cd $GITHUB_WORKSPACE/hcp-sdk-go
+          git config user.name "GitHub Actions"
+          git config user.email "<>"
+          service_branch_exists="$(git ls-remote --heads origin auto-update-${{ github.event.inputs.service }}-specs)"
```

Comment:
repeat of [this comment](https://github.com/hashicorp/cloud-api/pull/160/files#r882801750), but let's create a branch name variable to ensure consistency.